### PR TITLE
heatmap and ui design

### LINKED ISSUE #25: Section 2: Chart Box
Proposed charts:
- [x] Histogram of prices in the given country and date range @WilfHass 
- [x] Given a country, a time plot of the average booking price over the date range (line plot) @roanraina 
- [x] Histogram/bar chart of the busiest days over the date range @Mengjun74 
- [x] Heatmap @mrnabiz

### PR DESCRIPTION
**Pull request by mrnabiz**

### CODE REVIEW

File: app.R [hunk 1]
```diff
@@ -0,0 +1,270 @@
...
+  
+  # Controller row
+  fluidRow(
+    dateRangeInput(inputId = 'daterange',
+                   label = '🗓️ Date range:',
+                   start  = min(data$reservation_status_date),
+                   end = max(data$reservation_status_date),
+                   format = "mm/dd/yyyy"
+                   ),
+    selectInput(inputId = 'countires',
```
Comment (WilfHass) [inline app.R]: Is this meant to be "countires" as the inputID?

Comment (mrnabiz) [reply app.R]: yeah, sorry for the typo. I'll fix it now.

**Comment by WilfHass** [comment]
Also, since our app is probably going to start getting complicated, are we able to separate the UI and server elements as we didin the previous file structure?

**Comment by mrnabiz** [comment]
> Also, since our app is probably going to start getting complicated, are we able to separate the UI and server elements as we didin the previous file structure?

Yeah, I coincidentally deleted them last night but we need to separate them now.
Please don't approve the PR, we'll discuss it during the class.

**Review by WilfHass — APPROVED** [review]

**Pull request merged**